docs/src/main/sphinx/release/release-362.md

```diff
@@ -0,0 +1,58 @@
+# Release 362 (XX Sep 2021)
+
+## General
+
+* Add support for {func}`LISTAGG`. ({issue}`4835`)
+* Improve join performance. ({issue}`8974`)
+* Improve performance of queries that contain `DISTINCT` clause. ({issue}`8967`)
+* Improve query performance by reducing CPU overhead of repartitioning data across worker nodes. ({issue}`9102`)
+* Prevent failure when querying `system.materialized_views` and materialized view is dropped concurrently. ({issue}`9050`) 
+* Fix `ALTER SCHEMA ... SET AUTHORIZATION` to resolve roles using the catalog of the schema instead of the session catalog. ({issue}`9066`)
+* Fix query failure when query contains a cast from `varchar` to a shorter `char`. ({issue}`9036`)
+* Fix planning of recursive queries when the recursion the base plan or the recursion step plan produce duplicate outputs.
```

Comment:
Should there be a comma here?

```suggestion
* Fix planning of recursive queries when the recursion, the base plan or the recursion step plan produce duplicate outputs.
```

Reply:
needs one more comma.  

the base plan, or the